README.md

```diff
@@ -1 +1,226 @@
+
+Until this is properly fix in OpenShift, you will need to enable the RO endpoint for each of your nodes.
+
+For each of your node's `node-config.yaml` you will need to run the following command to enable read only access:
+
+	cat >> node-config.yaml << DONE
+	kubeletArguments:
+ 	read-only-port:
+ 	- "10266"
+	DONE
+
+### Creating Persistent Storage
+
+The Cassandra database stores its data to persistent storage. For each Cassandra node you deploy, you will need a persistent volume with sufficient data available. You do not need to directly manage peristent volume claims as the deployer and templates will take care of that for you.
+
+Please see the [OpenShift documentation](https://docs.openshift.org/latest/architecture/additional_concepts/storage.html) for how to setup and configure persistent volumes.
+
+For example if you have a NFS server running on localhost with an exposed directory at `/persistent_storage/pv01`, the following command will generate a 10 gigabyte persistent volume:
+
+	oc create -f - <<API
+	apiVersion: v1
+	kind: PersistentVolume
+	metadata:
+	  name: my_pv
+	spec:
+	  capacity:
+	    storage: 10Gi
+	  accessModes:
+	    - ReadWriteOnce
+	    - ReadWriteMany
+	  persistentVolumeReclaimPolicy: Recycle
+	  nfs:
+	    server: localhost
+	    path: /persistent_storage/pv01
+	API
+
+
+## Deploying
+
+
+### Create a Metrics Project
+
+To prevent unwanted components from accessing the metric's persistent volume claim, it is highly recommended to deploy all the metric components within their own project. Management of the metric components is also easier if they are separated in their own project as well.
+
+For the instructions presented here we are going to do so in a project named `metrics`.
+
+To create a new project called `metrics` you will need to run the following command:
+
+	oc new-project metrics
+
+### Create the Deployer Service Account
+
+A pod is used to setup, confiure and deploy all the various metric components. This deployer pod is run under the `metrics-deployer` service account.
+
+To create the metrics deployer service account, the following command can be run:
+
+	oc create -f - <<API
+	apiVersion: v1
+	kind: ServiceAccount
+	metadata:
+	  name: metrics-deployer
+	secrets:
+	- name: metrics-deployer
+	API
+
+### Service Account Permissions
+
+Note: the following commands assumes you are running in the 'metric' project, if you are running in another project, your service account will need to be updated the service account accordingly. The format follows the following format  `system:serviceaccount:$PROJECT_NAME:$SERVICE_ACCOUNT_NAME`
+
+#### Metrics Deployer Service Account
+
+In order to deploy components within the project, the `metrics-deployer` service account needs to be granted the 'edit' permission.
+
+This can be accomplished by running the following command:
+
+	openshift admin policy add-role-to-user edit \
```

Comment:
Ok, good point. We should use the 'oadm' command instead of 'openshift admin' as the user running it may not be running locally.